lib/OpenQA/Worker.pm

```diff
@@ -579,9 +545,7 @@ sub _inform_webuis_before_stopping {
 }
 
 # stops the current job and (if there is one) and terminates the worker
-sub stop {
-    my ($self, $reason) = @_;
-
+sub stop ($self, $reason) {
```

Comment:
It should be ok to not pass a `$reason`.

Reply:
ok. I guess I will need to exclude further going signatures changes from this "fix" PR